packages/peregrine/lib/talons/MegaMenu/useMegaMenuItem.js

```diff
@@ -6,9 +6,12 @@ export const useMegaMenuItem = props => {
     const [isFocused, setIsFocused] = useState(false);
     const isActive = category.id === activeCategoryId;
 
-    const handleCloseSubMenu = useCallback(() => {
-        setIsFocused(false);
-    }, [setIsFocused]);
+    const handleMenuItemFocus = useCallback(
+        (state = false) => {
```

Comment:
Instead of using a state parameter here, I would suggest adding separate handlers for Blur state (`setisFocused(false)`) and Focus state (`setIsFocused(true)`).

Reply:
Fixed in commit #6088fa7